### ISSUE TITLE
Starting a server clears the GamePassword field

### ISSUE THREAD

**Issue by AlienXAXS** [3RaGaming/Web_Control#49]
Steps To Reproduce:
1) Set the Game Password in the Config section
2) Start the game server
3) Notice that the Game Password no longer exists.

**Comment by stuudmuffin** [comment]
Checking

**Comment by stuudmuffin** [comment]
I feel this may be a Factorio bug. The web control is saving the config file properly, but the server appears to be deleting it. Even tried manually changing via ssh. It is possible they have changed how server passwords work. Checking change logs.

**Comment by stuudmuffin** [comment]
Submitted bug report https://forums.factorio.com/viewtopic.php?f=7&t=48534

**Comment by AlienXAXS** [comment]
Weirdly it does not happen on my CentOS box... will check my Ubuntu box to just double check the findings.

**Comment by AlienXAXS** [comment]
I have modified the settings file, added a game password.

Ran:
'/usr/share/factorio/0.15.13/bin/x64/factorio --start-server-load-latest --port 34291 -c /var/www/factorio/server1/config/config.ini --server-setting /var/www/factorio/server1/server-settings.json'

Closed the server, game password still exists...

**Comment by zackman0010** [comment]
I remember this happening to me a while back. I thought I was just doing something wrong at the time.

**Comment by stuudmuffin** [comment]
Found the issue. I hadn't used a password in so long or I might have cause it sooner. We used to only have two server settings on the web gui, name, and password. both were checked and updated at server launch. The password was blank by default. After moving the password change to a dedicated page, the old code was still in place to attempt to set the password on server star. 

I have removed this, and now server password works as it should.
